docs/content/basics.md

```diff
@@ -1,3 +1,52 @@
 # Basic usage
 
-TODO: show how to use https://github.com/mhausenblas/polly/blob/main/examples/basic/example.cue
+We have a very basic example in [examples/basic/]( https://github.com/mhausenblas/polly/blob/main/examples/basic) that you can use to explore polly packages.
+This example contains multiple signals (in this case PromQL queries), a dataface, and in the end generates a Grafana dashboard with those lower level constructs.
+
+### Evaluating
+
+Because the example comes with parameterized parameters (params) we cannot straight export JSON or  YAML, but instead can evaluate the example. This means that CUE will evaluate as much as possible of the end result, still leaving a few places without the final parameters/strings.
+
+From the root of the repository you can evaluate by running:
+
+```bash
+cue eval ./examples/basic
+```
+
+*Note: If any of the constraints aren't fulfilled this step will fail and let you know that something is wrong.*
+
+### Exporting
+
+To export you need to insert actual parameters to the polly package for it to be able to actually export real YAML. Let's give the example some concrete params:
+
+```diff
+ 		{
+ 			name: "NumCpu"
+ 			lang: "promql"
+-			params: {job: string, instance: string}
++			params: {job: "job", instance: "instance"}
+ 			query: "count without (cpu) (count without (mode) (node_cpu_seconds_total{job=\"\(params.job)\", instance=\"\(params.instance)\"}))"
+ 		},
+ 		// Amount of memory currently in use
+ 		{
+ 			name: "MemoryUtilization"
+ 			lang: "promql"
+-			params: {job: string, instance: string}
++			params: {job: "job", instance: "instance"}
```

Comment:
Probably better to use values here that are realistic values for the job and instance labels, rather than the strings "job" and "instance". (Those aren't realistic, are they?)